go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cert-manager/approver-policy
 
-go 1.20
+go 1.21.1
```

Comment:
Why change this? And why not the latest patch version of Go?

Reply:
I reverted this. I made it so the go version is no longer coupled to the go.mod file.